areaOfInterest/areas-of-interest.ts

```diff
@@ -0,0 +1,103 @@
+                //if thumbnail pictures disabled show list of areas as black buttons
+                if (noPic) {
+                    zoneList +=
+                        `<li class="rv-basemap-list-item" style="height: 50px;">
+                                      <div style="position: relative; display: flex; align-items: center; justify-content: center; overflow: hidden; height: 100%">
+                                          <div style="max-height: 50px; max-width: 350px; text-align: center;"></div>
+                                          <div class="rv-basemap-footer"><span>${zone.title}</span></div>
+                                              <button class="rv-body-button rv-button-square md-button md-ink-ripple"
+                                                  type="button" aria-label="bookmark" onclick="setExtent(${zone.xmin}, ${zone.xmax}, ${zone.ymin}, ${zone.ymax}, ${map.id})">
+                                                  <div class="md-ripple-container" style=""></div>
+                                              </button>
+                                          </div>
+                                  </li>`;
+                }
+                //else show list of areas with thumbnails
+                else {
+                    zoneList +=
+                        `<li class="rv-basemap-list-item" style="height: 175px;">
+                                      <div style="position: relative; display: flex; align-items: center; justify-content: center; overflow: hidden; height: 100%">
+                                          <div style="max-height: 175px; max-width: 350px; text-align: center;"><img alt="" src="${img}"/></div>
+                                          <div class="rv-basemap-footer"><span style="text-overflow:ellipses">${zone.title}</span></div>
+                                              <button class="rv-body-button rv-button-square md-button md-ink-ripple"
+                                                  type="button" aria-label="bookmark" onclick="setExtent(${zone.xmin}, ${zone.xmax}, ${zone.ymin}, ${zone.ymax}, ${map.id})">
+                                                  <div class="md-ripple-container" style=""></div>
+                                              </button>
+                                          </div>
+                                  </li>`;
+                }
+            }
+        }
+        //delete previously added area of interest panel (useful for language switch)
+        if (document.getElementById('area-of-interest')) {
+            map.deletePanel('area-of-interest');
+        }
+        //creating Panel + opening
+        let height = $('#' + map.id).height();
+        let panel0 = map.createPanel('area-of-interest');
+        if (height) {
+            panel0.position([-10, -10], [290, height - 52]);
+        }
+        $('div').removeClass('hidden');
```

Comment:
Calling `panel.open()` should handle the visibility change for you.

Reply:
Done